### PR TITLE
Fix for https://issues.redhat.com/browse/MODCLUSTER-734

### PR DESCRIPTION
**Pull request by jfclere**
https://issues.redhat.com/browse/MODCLUSTER-734
Add ModProxyClusterHCTemplate directive to allow a template health check to be added to each mod_proxy_cluster worker.
The format of the ModProxyClusterHCTemplate is the same than the mod_proxy_hcheck ProxyHCTemplate directive.

### CODE REVIEW

File: native/mod_proxy_cluster/mod_proxy_cluster.c [hunk 1]
```diff
@@ -1942,10 +2058,22 @@ static int proxy_cluster_post_config(apr_pool_t *p, apr_pool_t *plog,
                      "Failed to register watchdog callback (%s)",
                      LB_CLUSTER_WATHCHDOG_NAME);
         return !OK;
-    }
+        }
```
Comment (jajik) [inline native/mod_proxy_cluster/mod_proxy_cluster.c:2061]: Double indent instead of single.

File: native/mod_proxy_cluster/mod_proxy_cluster.c [hunk 2]
```diff
@@ -1942,10 +2058,22 @@ static int proxy_cluster_post_config(apr_pool_t *p, apr_pool_t *plog,
...
+
+/* pre_config for the health check part */
+static int proxy_cluster_pre_config(apr_pool_t *pconf, apr_pool_t *plog,
+                               apr_pool_t *ptemp)
+{
+    set_worker_hc_param_f = APR_RETRIEVE_OPTIONAL_FN(set_worker_hc_param);
     return OK;
 }
 
+
```
Comment (jajik) [inline native/mod_proxy_cluster/mod_proxy_cluster.c:2076]: Additional blank line.

File: native/mod_proxy_cluster/mod_proxy_cluster.c [hunk 3]
```diff
@@ -3070,6 +3199,39 @@ static const char*cmd_proxy_cluster_cache_shared_for(cmd_parms *cmd, void *dummy
     }
     return NULL;
 }
+static const char*cmd_proxy_cluster_proxyhctemplate(cmd_parms *cmd, void *dummy, const char *arg)
```
Comment (jajik) [inline native/mod_proxy_cluster/mod_proxy_cluster.c:3202]: Missing space after `static const char` (return type).

**Review by jajik — APPROVED** [review]
A few minor notes but it seems ok to me.

File: native/mod_proxy_cluster/mod_proxy_cluster.c (hunk 1 above)
Comment (jfclere) [reply native/mod_proxy_cluster/mod_proxy_cluster.c:2061]: Fixed

**Pull request merged**